.github/dependabot.yml

```diff
@@ -19,6 +19,16 @@ updates:
       timezone: "Europe/Amsterdam"
     reviewers:
       - "infonl/dimpact"
+    ignore:
+      # Ignore updates for these packages because it is needed for our current WildFly version (26.x)
```

Comment:
is this the only way to ignore these files? Seems convoluted...

Reply:
It's temporary, we have wildfly update on the backlog PZ-592

Reply:
No it's not temporary? We will always have these `providedCompile` dependencies. And they can never be upgraded by Dependabot. That's why we want to configure them here.

Reply:
Why is this convoluted you think? I think it's quite nice to do it this way. See: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

Reply:
Okay it's not temporary no. But it might need to be changed when wildlfy is updated

Reply:
Absolutely yes. Remove the 'might'. ;-) I expect that for every WildFly upgrade we will need to manually updated these deps.